frontend/app/types.d.ts

```diff
@@ -4,7 +4,7 @@ import type gcweb from '../public/locales/en/gcweb.json';
 import type index from '../public/locales/en/index.json';
 import type personalInformation from '../public/locales/en/personal-information.json';
 import type updatePhoneNumber from '../public/locales/en/update-phone-number.json';
-import type viewLetters from '../public/locales/en/view-letters.json';
+import type Letters from '../public/locales/en/letters.json';
```

Comment:
To follow existing conventions:

```suggestion
import type letters from '../public/locales/en/letters.json';
```